Dockerfiles/in-toto-container.Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM python:latest
```

Comment:
Now necessarily a fan of the `Dockerfiles` directory, neither the name, nor the capitalization - do we think we will have multiple Dockerfiles in the future to justify having a separate directory?